### ISSUE TITLE
Over used 3D printing supports

### ISSUE THREAD

**Issue by kaklik** [ThunderFly-aerospace/TF-SIMPLEPLATFORM#14]
Mnoho dílů platformy je navrženo tak, že k vytištění potřebuje použití 3D tištěných podpor. Takové řešení výrazně komplikuje výrobu platformy i její opravitelnost.

**Comment by slimonslimon** [comment]
Při tisku jedné platformy ročně je zbytečné se tím zabývat. 

Až spotřeba dílů stoupne, bude na čase rozmýšlet kompletní renávrh dílů, které jsou komplikované tak aby se skládali z více kusů, či vypadali úplně jinak.

**Comment by kaklik** [comment]
> Při tisku jedné platformy ročně je zbytečné se tím zabývat.
> 
> Až spotřeba dílů stoupne, bude na čase rozmýšlet kompletní renávrh dílů, které jsou komplikované tak aby se skládali z více kusů, či vypadali úplně jinak.

Ano, tento argument chápu, ale stejně by jsi si mohl ty podpory zkusit příště vydloubat ty.. ;)

**Comment by slimonslimon** [comment]
To jsem měl v plánu, jen mě někdo předběhl ;-) 

Tomu celkem věřím, že některé díly budou potřebovat úpravu pro lepší použitelnost....
